src/libraries/System.Net.Security/tests/FunctionalTests/TlsFrameHelperTests.cs

```diff
@@ -94,7 +97,7 @@ public static IEnumerable<object[]> InvalidClientHelloData()
             }
         }
 
-        public static IEnumerable<object[]> InvalidClientHelloDataTruncatedBytes()
+        public static IEnumerable<Tuple<int, byte[]>> InvalidClientHelloDataTruncatedBytes()
```

Comment:
Nit, as a style thing, we seem (I think?) to most often use the `(int, byte[])` syntax (possibly with names) not the old `Tuple<X, Y>` syntax.